src/timeline-manager.tsx

```diff
@@ -131,8 +131,12 @@ class TimelineManager {
     }
   };
 
+  private _getProgressBarEl() {
+    return document.getElementsByClassName(style.progressBar)[0];
+  }
+
   private _addSegmentToSeekbar() {
-    const progressBarEl = document.getElementsByClassName(style.progressBar)[0];
+    const progressBarEl = this._getProgressBarEl();
     if (progressBarEl && !progressBarEl.classList.contains(style.chapters)) {
```

Comment:
Instead of - (progressBarEl && !progressBarEl.classList.contains(style.chapters))
Just - (progressBarEl?.classList.contains(style.chapters))

Reply:
oh yeah i forgot to change what Tzipi did

Reply:
actually- here it's not the same. the vaidation should return true only if the element exists and doesn't have the chapters class.